src/services/messengerSync/producer.js

```diff
@@ -0,0 +1,80 @@
+const { Configuration } = require('@schul-cloud/commons');
+const { createChannel } = require('../../utils/rabbitmq');
+
+const QUEUE_INTERNAL = Configuration.get('RABBITMQ_MATRIX_QUEUE_INTERNAL');
```

Comment:
For reasons of change the QUEUE_INTERNAL during testing by updating the configuration value, use Configuration.get() directly